lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -65,25 +79,20 @@ def code
       end
 
       # REQUIRED - This is the http status code.
-      # unless you've specified that you want the status code to be something other
-      # then 400 explicitly it will default to 400. IT WILL NOT DEFAULT to whatever
-      # was provided by the backend service, because the backend service response
-      # might not always be relevant
       def status
-        i18n_data[:status].presence || 400
+        i18n_data[:status] || 400
       end
 
       # OPTIONAL - This is the detail or message that is rendered in JSON response
       # Not providing detail will render a detail the same as title, 'Operation failed'
-      # NOTE: in the future, detail will only work via i18n, not the value from response_values
       def detail
-        i18n_data[:detail].presence || response_values[:detail]
+        i18n_data[:detail] || service_response[:detail]
```

Comment:
similarly here, i'm not too keen on blindly passing the value from service_response, but I do think it's useful. If i18n detail is nil, then service_response will work, if it has a value, then it will use that one instead. If both of them are nil, then when it gets serialized it will default to title in ErrorSerializer. But is that clear necessarily from looking at this? Probably not. How can I make it more clear?